src/Microsoft.VisualBasic/tests/OperatorsTests.cs

```diff
@@ -5456,11 +5456,11 @@ private static void AssertEqual(object expected, object actual)
         {
             if (expected is double expectedDouble && actual is double actualDouble)
             {
-                Assert.Equal(expected.ToString(), actual.ToString());
+                Assert.Equal(expectedDouble, actualDouble, 10);
```

Comment:
This doesn't seem right, on all platforms the results for these simple operations should be deterministic.

We were previously correct for 15 digits, so I would expect the same here (at a minimum). Ideally we would be comparing to the full 17 digits, however (or just doing a bit comparison).

Reply:
@hughbe, can we simply use `Assert.Equal(expected, actual)` in all cases?

Reply:
@hughbe, can we use `Assert.Equal()`? Thanks.

Reply:
This comment is out-of-date